diff -up aircrack-ng-1.6/lib/radiotap/radiotap.h.omv~ aircrack-ng-1.6/lib/radiotap/radiotap.h
--- aircrack-ng-1.6/lib/radiotap/radiotap.h.omv~	2021-01-01 04:49:15.506036526 +0100
+++ aircrack-ng-1.6/lib/radiotap/radiotap.h	2021-01-01 04:49:29.167202022 +0100
@@ -56,7 +56,7 @@ struct ieee80211_radiotap_header {
 	 * @it_present: (first) present word
 	 */
 	uint32_t it_present;
-} __packed;
+} __attribute__((packed));
 
 /* version is always 0 */
 #define PKTHDR_RADIOTAP_VERSION	0
diff -up aircrack-ng-1.6/src/aireplay-ng/aireplay-ng.c.omv~ aircrack-ng-1.6/src/aireplay-ng/aireplay-ng.c
diff -up aircrack-ng-1.6/src/airodump-ng/airodump-ng.c.omv~ aircrack-ng-1.6/src/airodump-ng/airodump-ng.c
--- aircrack-ng-1.6/src/airodump-ng/airodump-ng.c.omv~	2021-01-01 04:52:29.831390960 +0100
+++ aircrack-ng-1.6/src/airodump-ng/airodump-ng.c	2021-01-01 04:52:36.554472427 +0100
@@ -92,8 +92,8 @@
 #include "radiotap/radiotap_iter.h"
 
 struct devices dev;
-uint8_t h80211[4096] __attribute__((aligned(16)));
-uint8_t tmpbuf[4096] __attribute__((aligned(16)));
+extern uint8_t h80211[4096] __attribute__((aligned(16)));
+extern uint8_t tmpbuf[4096] __attribute__((aligned(16)));
 
 static const unsigned char llcnull[] = {0, 0, 0, 0};
 
diff -up aircrack-ng-1.6/src/airventriloquist-ng/airventriloquist-ng.h.omv~ aircrack-ng-1.6/src/airventriloquist-ng/airventriloquist-ng.h
--- aircrack-ng-1.6/src/airventriloquist-ng/airventriloquist-ng.h.omv~	2021-01-01 04:47:49.035989089 +0100
+++ aircrack-ng-1.6/src/airventriloquist-ng/airventriloquist-ng.h	2021-01-01 04:48:10.287246496 +0100
@@ -50,7 +50,7 @@ struct net_hdr
 	uint8_t nh_type;
 	uint32_t nh_len;
 	uint8_t nh_data[0];
-} __packed;
+} __attribute__((packed));
 
 struct llc_frame
 {
diff -up aircrack-ng-1.6/src/easside-ng/easside-ng.c.omv~ aircrack-ng-1.6/src/easside-ng/easside-ng.c
