--- aircrack-ng-1.1/common.mak.make_makeup~	2011-07-07 23:42:32.000000000 +0200
+++ aircrack-ng-1.1/common.mak	2011-07-07 23:56:32.510111869 +0200
@@ -31,16 +31,16 @@ COMMON_CFLAGS   += -DCYGWIN
 endif
 
 ifeq ($(SQLITE), true)
-    COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+    COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
 else
     ifeq ($(sqlite), true)
-        COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+        COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
     else
         ifeq ($(SQLITE), TRUE)
-            COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+            COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
         else
             ifeq ($(sqlite), TRUE)
-                COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+                COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
             endif
         endif
     endif
@@ -66,14 +66,14 @@ AR		= $(TOOL_PREFIX)ar
 REVISION	= $(shell $(AC_ROOT)/evalrev)
 REVFLAGS	= -D_REVISION=$(REVISION)
 
-OPTFLAGS        = -D_FILE_OFFSET_BITS=64
+OPTFLAGS        = $(shell getconf  LFS_CFLAGS)
 CFLAGS          ?= -g -W -Wall -O3
 CFLAGS          += $(OPTFLAGS) $(REVFLAGS) $(COMMON_CFLAGS)
 
 prefix          = /usr/local
 bindir          = $(prefix)/bin
 sbindir         = $(prefix)/sbin
-mandir          = $(prefix)/man/man1
+mandir          = $(datadir)/man/man1
 datadir         = $(prefix)/share
 docdir          = $(datadir)/doc/aircrack-ng
 libdir		= $(prefix)/lib
--- aircrack-ng-1.1/manpages/Makefile.make_makeup~	2011-07-07 23:42:31.000000000 +0200
+++ aircrack-ng-1.1/manpages/Makefile	2011-07-07 23:52:25.237294177 +0200
@@ -47,13 +47,13 @@ all:
 	@echo Nothing to do
 	
 install: uninstall
-	install -d $(DESTDIR)$(mandir)
-	install -m 644 $(MP_FILES) $(DESTDIR)$(mandir)
+	install -d $(DESTDIR)$(mandir)/man1
+	install -m 644 $(MP_FILES) $(DESTDIR)$(mandir)/man1
 
 uninstall:
-	@-$(foreach CUR_MP,$(MP_FILES), rm -f $(DESTDIR)$(mandir)/$(CUR_MP); echo rm -f $(DESTDIR)$(mandir)/$(CUR_MP); ) 
-	-rm -f $(DESTDIR)$(mandir)/airolib-ng.1
-	-rm -f $(DESTDIR)$(mandir)/wesside-ng.1
-	-rm -f $(DESTDIR)$(mandir)/tkiptun-ng.1
-	-rm -f $(DESTDIR)$(mandir)/buddy-ng.1
-	-rm -f $(DESTDIR)$(mandir)/easside-ng.1
+	@-$(foreach CUR_MP,$(MP_FILES), rm -f $(DESTDIR)$(mandir)/man1/$(CUR_MP); echo rm -f $(DESTDIR)$(mandir)/man1/$(CUR_MP); ) 
+	-rm -f $(DESTDIR)$(mandir)/man1/airolib-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/wesside-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/tkiptun-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/buddy-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/easside-ng.1
